File: 0283-move-zeroes/0283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& arr) {
        int i = 0;
        int j = 0;
        for(j = 0;j<arr.size();j++){
            if(arr[j]!=0){
                swap(arr[i],arr[j]);
                i++;
            }
        }
    }
};

File: 0283-move-zeroes/0283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int i = 0;
        int j = 0;
        for(j = 0;j<nums.length;j++){
            if(nums[j]!=0){
                int x = nums[i];
                nums[i] = nums[j];
                nums[j] = x;
                i++;
            }
        }
    }
}